llvm/lib/Transforms/IPO/WholeProgramDevirt.cpp

```diff
@@ -168,6 +168,14 @@ static cl::list<std::string>
                       cl::desc("Prevent function(s) from being devirtualized"),
                       cl::Hidden, cl::CommaSeparated);
 
+/// If this value is other than 0, the devirt module pass will stop
```

Comment:
I believe the way the code is written, value 0 is not special and will cause no devirts to happen - assuming the option was specified (i.e. getNumOccurrences() > 0). And in fact we probably want a way to say "do 0 devirtualizations".

Reply:
I updated the comment, and yes it makes sense to do 0 devirtualization when this option is explicitly set to zero.